### ISSUE TITLE
Empty folder in file list

### ISSUE THREAD

**Issue by tim-gromeyer** [doxygen/doxygen#9683]
**Describe the bug**

In my file list is a empty folder.

**Expected behavior**
Remove the empty folder from the list.

**Screenshots**
![Empty_folder](https://user-images.githubusercontent.com/58736434/202717229-c487d090-c86c-4b55-a85d-1116258d8dc8.png)

**To Reproduce**
[example.tar.gz](https://github.com/doxygen/doxygen/files/10041836/example.tar.gz)
[example.zip](https://github.com/doxygen/doxygen/files/10041837/example.zip)

Unpack it and run doxygen in the example dir, **not in the docs dir!**

`doxygen docs/Doxyfile`

**Version**
Precompiled version.
`doxygen --version`: 1.9.4 (5d15657a55555e6181a7830a5c723af75e7577e2)

<br>

`lsb_release -a`:  
No LSB modules are available.
Distributor ID:	Linuxmint
Description:	Linux Mint 21
Release:	21
Codename:	vanessa

**Additional context**
It's not doxygen-awesome-css's fault!

**Comment by albert-github** [comment]
Note:
- the current doxygen version is 1.9.5 and also has support for dark style (see `HTML_COLORSTYLE`).
- there is no `src` directory in the archives, but as far as I can see you are here referring to the `docs` directory.

The `docs` directory is used for input but due to the fact that there are, relevant for doxygen, only markdown files present the directory becomes empty. Won't be easy to remove the empty directory references for all output formats, I'll mark it as enhancement.

**Comment by albert-github** [comment]
I revisited the problem and it looks like it is "created" due to the fact that `EXTRACT_ALL=YES` which indicates:
> If the EXTRACT_ALL tag is set to YES, doxygen will assume all entities in documentation are
documented, even if no documentation was available.

and a directory is also an entity (see also the `\dir` command), so the empty directory is a directory that should be be shown as well.
Directories that don't have any files in it are already filtered out during the determination of the files to be processed and are not shown either, even though they are documented by means of the `\dir` command. 

@doxygen is it correct that the directories that have no files in them but are documented are not shown?

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/10049329/example.tar.gz)

**Comment by tim-gromeyer** [comment]
> * the current doxygen version is 1.9.5 and also has support for dark style (see `HTML_COLORSTYLE`).

Yep, but doxygen-awesome-css just looks more modern. doxygen-awesome-css has a bug with doxygen 1.9.5(#9549) so I'm currently still using 1.9.4.

> * there is no `src` directory in the archives, but as far as I can see you are here referring to the `docs` directory.

I have probably taken a screenshot of the wrong window

> The `docs` directory is used for input but due to the fact that there are, relevant for doxygen, only markdown files present the directory becomes empty. Won't be easy to remove the empty directory references for all output formats, I'll mark it as enhancement.

Maybe I can remove the foder with some JS.

**Comment by tim-gromeyer** [comment]
I'm using following js code to solve the issue:

```js
const directory = document.querySelector('.directory');
const table = directory.querySelector('table');

for (const row of table.rows) {
  if (row.querySelector('.iconfclosed') && !row.querySelector('.arrow')) {
    // Entferne die Zeile
    row.remove();
  }
}
```

It tested it with doxygen 1.9.4.
